jack.py

```diff
@@ -92,6 +92,85 @@
     JackTransportStarting = 3,
     JackTransportNetStarting = 4,
 } jack_transport_state_t;
```

Comment:
Please add the declarations in the same order as in the original header files.
This avoids repeated declarations, like for example `jack_nframes_t`.